src/labels/CustomLabels.labels

```diff
@@ -573,6 +573,22 @@
         <value>omit</value>
     </labels>
     <labels>
+        <fullName>EPTaskDependency</fullName>
+        <categories>Engagement Plan, Error</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>EPTaskDependency</shortDescription>
+        <value>dependency recursion is bad</value>
```

Comment:
Ha, this error message was a joke but I like that you took it at face value. How about: "Engagement Plan Task dependencies cannot make recursive dependency loops." For these simple edits to text strings, you can actually edit inline in github and it will create a commit for you.